src/sentry/sentry_apps/tasks/sentry_apps.py

```diff
@@ -270,6 +270,28 @@ def _process_resource_change(
         ]
 
         for installation in installations:
+            try:
+                service_hook = ServiceHook.objects.get(
+                    organization_id=org.id,
+                    installation_id=installation.id,
+                )
```

Comment:
In the other query where we get `ServiceHook` we use `actor_id=installation.id`, why the change here?

Reply:
we talk about it [here](https://github.com/getsentry/sentry/pull/85502#discussion_r1982323739) but for `ServiceHooks` created as part of installing a sentry app, they set both `installation_id` & `actor_id` as the same `SentryAppInstallation` id so either would work. But I think using `installation_id` is clearer than `actor_id`

Reply:
Latest revision: 
using `actor_id` to be safe (e.g. we could have historic data from before when `installation_id` was set during sentry app creation).
I added logger.error() if the two are not matching to help us clean it up later.